packages/SwingSet/tools/bundleTool.js

```diff
@@ -14,7 +14,7 @@ export const makeNodeBundleCache = async (dest, options, loadModule) => {
       styles.dim.close,
     );
   };
-  return wrappedMaker(dest, { log, ...options }, loadModule);
+  return wrappedMaker(dest, { log, ...options }, loadModule, pid);
```

Comment:
And, this is a complication that I introduced that I should have addressed in Agoric before we had attempted to sync the repos.

@michaelfig has created some tools to make these aberrations more visible. I haven’t internalized them well enough in my own process to recommend a workflow for the rest of the team.